membership-attribute-service/app/wiring/AppLoader.scala

```diff
@@ -43,9 +45,36 @@ class MyComponents(context: Context)
     new BehaviourController(commonActions)
   )
 
+  val regularCorsPaths = Seq(
+    "/user-attributes/me/membership",
+    "/user-attributes/me/features",
+    "/user-attributes/me",
+    "/user-behaviour/capture",
+    "/user-behaviour/remove",
+    "/abandoned-cart/email"
+  )
+
+  val mmaPaths = Seq(
+    "/user-attributes/me/mma-digitalpack",
+    "/user-attributes/me/mma-monthlycontribution",
+    "/user-attributes/me/mma-membership",
+    "/user-attributes/me/mma-paper")
+
+  val mmaUpdatePaths = Seq(
+    "/user-attributes/me/membership-update-card",
+    "/user-attributes/me/digitalpack-update-card",
+    "/user-attributes/me/paper-update-card",
+    "/user-attributes/me/contribution-update-card",
+    "/user-attributes/me/cancel-regular-contribution",
+    "/user-attributes/me/contribution-update-amount")
```

Comment:
I'm not 100% convinced with having to specify all the endpoints we want to enable CORS here individually, It makes it easier to forget to add new endpoints we might add.

If the endpoints were organized in a way we could just have a couple of general prefixes here it would be better, but sadly they are not

Reply:
Yes I'm not sure why we start almost everything with 'user-attributes/me'...